### PR TITLE
Add 'none' Condition for Convenient Testing in Package Update Logic

### PR DESCRIPTION
**Pull request by RoyCai7**
- Related ticket: https://progress.opensuse.org/issues/169999
- Needles: N/A
- Verification run: [Package is none](http://openqa.qam.suse.cz/tests/81639)
                               [ another VR test when package is none](http://openqa.qam.suse.cz/tests/81569#step/kernel/1) please ignore failure afterwards.

### CODE REVIEW

File: lib/main_common.pm [hunk 1]
```diff
@@ -2387,10 +2387,11 @@ sub set_mu_virt_vars {
     $BUILD =~ /^:(\d+):([^:]+)$/im;
 
     die "BUILD value is $BUILD, but does not match required format." if (!$2);
-
     my $_pkg = $2;
     my $_update_package = '';
-    if ($_pkg =~ /qemu|xen|virt-manager|libguestfs|libslirp|open-vm-tools/) {
+    if ($_pkg =~ /none/) {
```
Comment (alice-suse) [inline lib/main_common.pm:2392]: Will be good to add a comment to explain purpose of none -- facilitate devel test with no real incident?

**Review by alice-suse — APPROVED** [review]
LGTM
